files/maven_settings.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <mirrors>
+    <mirror>
+      <mirrorOf>external:*</mirrorOf>
+      <name>remote-repos</name>
+      <url>https://artifactory.automate.datapipe.net:443/remote-repos</url>
```

Comment:
I'd prefer to externalize it as well, but I don't believe it is possible in this case...